models/PatientSymptomEntry.js

```diff
@@ -0,0 +1,52 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const PatientEntrySchema = new Schema({
+
+   /* patientFullName:
+     {
+        firstName: {type: String,rquired: true },
+        lastName: {type: String,rquired: true },
+      },*/
+      patientFullName:
+       {type: String,rquired: true },
+       
+  form: {
+       date: {type: Date,rquired: true },
+    symptom1:{type: Boolean,rquired: true },
+    symptom2: {type: Boolean,rquired: true },
+    symptom3: {type: Boolean,rquired: true },
+             
+    symptom4: {type: Boolean,rquired: true },
```

Comment:
Same as above in the model symptoms are booleans and the data being sent from front end is string.  Also this model and the above model look very similar, are they both needed.